cmd/m3db-operator/main.go

```diff
@@ -26,32 +26,42 @@ import (
 	"os"
 	"os/signal"
 	"runtime"
-	"sync"
 	"syscall"
+	"time"
 
 	"github.com/m3db/m3/src/query/util/logging"
 	clientset "github.com/m3db/m3db-operator/pkg/client/clientset/versioned"
+	informers "github.com/m3db/m3db-operator/pkg/client/informers/externalversions"
 	"github.com/m3db/m3db-operator/pkg/controller"
 	"github.com/m3db/m3db-operator/pkg/k8sops"
+	"github.com/m3db/m3db-operator/pkg/m3admin/namespace"
+	"github.com/m3db/m3db-operator/pkg/m3admin/placement"
 
 	"go.uber.org/zap"
 	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	kubeinformers "k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+const (
+	// Informers will resync on this interval
+	_informerSyncDuration = 30 * time.Second
+)
+
 var (
-	appVersion  = "0.0.1"
-	kubeCfgFile string
-	masterHost  string
-	logrusLevel string
+	appVersion      = "0.0.1"
+	kubeCfgFile     string
+	masterHost      string
+	coordinatorAddr string
 )
 
 func init() {
 	flag.StringVar(&kubeCfgFile, "kubecfg-file", "", "Location of kubecfg file for access to kubernetes master service; --kube_master_url overrides the URL part of this; if neither this nor --kube_master_url are provided, defaults to service account tokens")
 	flag.StringVar(&masterHost, "masterhost", "http://127.0.0.1:8001", "Full url to k8s api server")
+	flag.StringVar(&coordinatorAddr, "coordinator", "", "override coordinator address if running out-of-cluster")
```

Comment:
Just a side note, in my other [PR](https://github.com/m3db/m3db-operator/pull/13) I switch from flags to environment variables. Not sure if we want to go down this route here.